pkg/utils/accessLogUtils.go

```diff
@@ -4,21 +4,35 @@ import (
 	"fmt"
 	"os"
 	"strings"
+	"time"
 
 	"github.com/siglens/siglens/pkg/common/dtypeutils"
 	log "github.com/sirupsen/logrus"
 )
 
+func DeferableAddAccessLogEntry(startTime time.Time, endTimeFunc func() time.Time, user string,
+	uri string, requestBody string, statusCodeFunc func() int, fileName string) {
+
+	data := dtypeutils.AccessLogData{
+		TimeStamp:   startTime.Format("2006-01-02 15:04:05"),
+		UserName:    user,
+		URI:         uri,
+		RequestBody: requestBody,
+		StatusCode:  statusCodeFunc(),
+		Duration:    endTimeFunc().Sub(startTime).Milliseconds(),
+	}
+	AddAccessLogEntry(data, fileName)
+}
+
+// Write to access.log in the following format
+// timeStamp <logged-in user> <request URI> <request body> <response status code> <elapsed time in ms>
 func AddAccessLogEntry(data dtypeutils.AccessLogData, fileName string) {
 	logFile, err := os.OpenFile(fileName, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
 	if err != nil {
 		log.Errorf("Unable to write to access.log file, err=%v", err)
 	}
 	defer logFile.Close()
-	// Do not log websocket connections
-	if data.StatusCode == 101 {
-		return
-	}
```

Comment:
Why did we have this here? Should I do something other than deleting it so that we can still log the UI queries?

Reply:
during the websocket based queries, we were getting lot of lines getting logged, because they come over 101. Hence this code was disabled.